lib/reqTemplate.js

```diff
@@ -4,7 +4,7 @@ const URI = require('./uri');
 const TAssembly = require('tassembly');
 const expressionCompiler = require('template-expression-compiler');
 const utils = require('./utils');
-const TimeUUID = require('cassandra-uuid').TimeUuid;
+const Uuidv1 = require('uuid/v1');
```

Comment:
Let's be consistent with how we name it across repos. Lowercase it please.

Reply:
Change to lowercase